app/server/ruby/lib/sonicpi/synths/synthinfo.rb

```diff
@@ -3498,6 +3498,221 @@ def specific_arg_info
+            :modulatable => true
+          },
+          :quint =>
+          {
+            :doc => "The drawbar for the tonewheel creating a sound component 3/2 of the frequency of the base tone",
+            :validations => [v_between_inclusive(:quint, 0, 8)],
+            :modulatable => true
+          },
+          :fundamental =>
+          {
+            :doc => "The drawbar for the tonewheel creating the base tone component. If you turn vibrato off and set just this drawbar to 8 and all the others to 0, you basically get a sine tone for the note to be played.",
+            :validations => [v_between_inclusive(:fundamental, 0, 8)],
+            :modulatable => true
+          },
+          :oct =>
+          {
+            :doc => "The drawbar for the tonewheel creating a sound component one octave above the base tone, i. e. twice its frequency",
+            :validations => [v_between_inclusive(:oct, 0, 8)],
+            :modulatable => true
+          },
+          :nazard =>
+          {
+            :doc => "The drawbar for the tonewheel creating a sound component 3 times the frequency of the base tone",
+            :validations => [v_between_inclusive(:nazard, 0, 8)],
+            :modulatable => true
+          },
+          :blockflute =>
+          {
+            :doc => "The drawbar for the tonewheel creating a sound component 2 octaves above the base tone, i. e. 4 times its frequency.",
+            :validations => [v_between_inclusive(:blockflute, 0, 8)],
+            :modulatable => true
+          },
+          :tierce =>
+          {
+            :doc => "The drawbar for the tonewheel creating a sound component 5 times the frequency of the base tone.",
+            :validations => [v_between_inclusive(:tierce, 0, 8)],
+            :modulatable => true
+          },
+          :larigot =>
+          {
+            :doc => "The drawbar for the tonewheel creating a sound component 6 times the frequency of the base tone.",
+            :validations => [v_between_inclusive(:larigot, 0, 8)],
+            :modulatable => true
+          },
+          :sifflute =>
+          {
+            :doc => "The drawbar for the tonewheel creating a sound component 3 octaves above the base tone, i. e. 8 times its frequency.",
+            :validations => [v_between_inclusive(:sifflute, 0, 8)],
+            :modulatable => true
+          },
+          :rs_freq =>
+          {
+            :doc => "Rotation frequency of the rotary speaker in Hertz. The tonewheel organ's rotary speaker affects sound in (at least) 3 ways: The frequency changes due to a Doppler effect, so that the pitch oscilates around the base frequency, the note the synth is playing. The amplitude and hence the perceived loudness change. When the horns rotate, they sound louder when they point towards the listener. The pan changes: When the horns point sideways, they sound louder on the the side they point to.
```

Comment:
```suggestion
            :doc => "Rotation frequency of the rotary speaker in Hertz. The tonewheel organ's rotary speaker affects sound in (at least) 3 ways: The frequency changes due to a Doppler effect, so that the pitch oscillates around the base frequency, the note the synth is playing. The amplitude and hence the perceived loudness change. When the horns rotate, they sound louder when they point towards the listener. The pan changes: When the horns point sideways, they sound louder on the side they point to.
```